examples/model_selection/plot_learning_curve.py

```diff
@@ -85,7 +85,7 @@ def plot_learning_curve(estimator, title, X, y, ylim=None, cv=None,
         plt.ylim(*ylim)
     plt.xlabel("Training examples")
     plt.ylabel("Score")
-    train_sizes, train_scores, test_scores = learning_curve(
+    train_sizes, train_scores, test_scores, _, _ = learning_curve(
```

Comment:
As should this

Reply:
Thanks for the catch! Reverted back in f3fe244